packages/serverless/src/integrations/aws_lambda.ts

```diff
@@ -0,0 +1,248 @@
+  ) {
+    this.awsContext = options.context;
+    this.timeoutWarning = options.timeoutWarning;
+    this.configuredTimeInMilliseconds = options.configuredTimeInMilliseconds as number;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setupOnce(_: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
+    // lambda-bootstrap
+    const lambdaBootstrap = require.main;
+
+    if (!this.awsContext && !lambdaBootstrap) {
+      return;
+    }
+
+    const nodeJs: string = 'Nodejs';
+
+    /** capturing this context */
+    const setupOnceContext = this;
+
+    /** rapid runtime instance */
+    const rapidRuntime = lambdaBootstrap?.children[0].exports;
+
+    /** handler that is invoked in case of unhandled and handled exception */
+    const invokedFunction = rapidRuntime.prototype.postInvocationError;
+
+    this.getCurrentHub = getCurrentHub;
+
+    const hub = this.getCurrentHub && this.getCurrentHub();
+
+    /** configured time */
+    this.configuredTimeInMilliseconds = this.awsContext.getRemainingTimeInMillis();
+
+    const timeoutWarningBuffer: number = 1500;
+    const millisToSeconds: number = 1000;
+
+    /** checking if configured Time In Milliseconds is greater than timeout Warning Buffer */
+    if (this.configuredTimeInMilliseconds > timeoutWarningBuffer) {
+      const configuredTimeInSec = this.configuredTimeInMilliseconds / millisToSeconds;
+      const configuredTime = Math.floor(configuredTimeInSec);
+      /** check timeout flag */
+      if (this.timeoutWarning === true) {
+        timeOutError(configuredTime);
+      }
+    }
+
+    /**
+     * This function is invoked when there is timeout error
+     * Here, we make sure the error has been captured by Sentry Dashboard
+     * and then re-raise the exception
+     * @param configuredTime  - configured time in seconds
+     * @hidden
+     */
+    function timeOutError(configuredTime: number): void {
+      setTimeout(() => {
+        /** return current event with event id  */
+        invokeEvent(uuid4());
+
+        /**
+         * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+         */
+        setParameters();
+
+        const error = new Error(
+          `WARNING : Function is expected to get timed out. Configured timeout duration = ${configuredTime +
+            1} seconds.`,
+        );
+        /** capturing the exception and re-directing it to the Sentry Dashboard */
+        hub.captureException(error);
+      }, configuredTime);
+    }
+
+    /**
+     * This function sets Additional Runtime Data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setAdditionalRuntimeData(scope: Scope): void {
+      scope.setContext('Runtime', {
+        Name: nodeJs,
+        // global.process.version return the version of node
+        Version: global.process.version,
+      });
```

Comment:
This should not be different than what we set from `@sentry/node`:

https://github.com/getsentry/sentry-javascript/blob/13b800e06601ad4e3fab9b553327b4b2bfcdd7a3/packages/node/src/handlers.ts#L268-L274

Reply:
The relevant code changes have been done